pkg/operator/etcdmemberscontroller/etcdmemberscontroller.go

```diff
@@ -156,10 +155,13 @@ func (c *EtcdMembersController) reportEtcdMembers() error {
 		// in the member list call and go to degraded with
 		// etcdserver: request timed out
 		_, _, updateErr := v1helpers.UpdateStatus(c.operatorClient, v1helpers.UpdateConditionFn(operatorv1.OperatorCondition{
-			Type:    "EtcdMembersAvailable",
-			Status:  operatorv1.ConditionFalse,
-			Reason:  "No quorum",
-			Message: fmt.Sprintf("%s members are available, %s have not started, %s are unhealthy", strings.Join(availableMembers, ","), strings.Join(notStartedMembers, ","), strings.Join(unhealthyMembers, ",")),
+			Type:   "EtcdMembersAvailable",
+			Status: operatorv1.ConditionFalse,
+			Reason: "No quorum",
+			Message: fmt.Sprintf("%s, %s, %s",
```

Comment:
I kind of expected you to have a method getMemberMessage that took the healthy, unhealthy, and not started and switch'd based on the numbers of each with clearer format strings.  combining clauses is always a bit awkward, single method is easier to test.